### PR TITLE
Fix Bug 1198773, Add Mark Mayo to steering committee page, and update titles on leadership page

### PR DESCRIPTION
**Pull request by schalkneethling**
@craigcook r?

### CODE REVIEW

**Comment by flodolo** [comment]
l10n is not a blocker, but I still need to udpate the strings for that page, hence the tag

**Comment by schalkneethling** [comment]
Thanks @flodolo meant to add that.

**Comment by craigcook** [comment]
A tentative r+ but waiting for confirmation on if we also need to add Jascha to the steering committee.

**Comment by jgmize** [comment]
> A tentative r+ but waiting for confirmation on if we also need to add Jascha to the steering committee.

We confirmed in the PR triage meeting that we don't need to wait on that, and I need to test a commit webhook that I just changed for a new CI server, so I'm going to go ahead and merge this now. Thanks @schalkneethling and @craigcook!

File: bedrock/foundation/templates/foundation/mocosc.html [hunk 1]
```diff
@@ -46,5 +46,6 @@ <h1 class="page-title">{{ _('About the Mozilla Corporation Steering Committee')
     <li><a href="https://blog.mozilla.org/press/bios/jim-cook/">Jim Cook, {{ _('Chief Finance Officer') }}</a></li>
     <li><a href="https://blog.mozilla.org/press/bios/denelle-dixon-thayer/">Denelle Dixon-Thayer, {{ _('Senior Vice President Business and Legal Affairs') }}</a></li>
     <li><a href="https://blog.mozilla.org/press/bios/david-slater/">David Slater, {{ _('Chief of Staff and SVP of Operations') }}</a></li>
+    <li><a href="https://blog.mozilla.org/press/bios/mark-mayo/">Mark Mayo, {{_('Senior Vice President, Firefox')}}</a></li>
```
Comment (l-hedgehog) [inline bedrock/foundation/templates/foundation/mocosc.html]: Should this list be in alphabetical order?

File: bedrock/mozorg/templates/mozorg/about/leadership.html [hunk 2]
```diff
@@ -159,7 +159,7 @@ <h3 id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{ _('Mozilla Corporat
           <img class="photo" itemprop="image" alt="" src="{{ static('img/mozorg/about/leadership/mark-mayo.jpg') }}">
           <span class="fn" itemprop="name">Mark Mayo</span>
         </a>
-        <span class="title" itemprop="jobTitle">{{ _('VP and General Manager, Firefox') }}</span>
+        <span class="title" itemprop="jobTitle">{{ _('Senior Vice President, Firefox') }}</span>
```
Comment (l-hedgehog) [inline bedrock/mozorg/templates/mozorg/about/leadership.html]: &lt;span class="note"&gt;{{ _('Steering Committee') }}&lt;/span&gt; here?

Should this be moved up to group with other SC members?

**Comment by l-hedgehog** [comment]
Also the "as of May, 2015" should be "as of August, 2015".

**Comment by craigcook** [comment]
@l-hedgehog You're correct, sorry I missed those details in my first pass. I was planning to take another pass at this PR and merge it earlier, but got caught up in other stuff and didn't have time. :weary:  I'll submit a follow-up PR momentarily to make those tweaks.

**Pull request merged**